app/components/view-epic-list/macro.test.js

```diff
@@ -0,0 +1,272 @@
+        data: [],
+        type: 'not-met',
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+        expect($('[data-test-id="some-data-identifier"]').length).toEqual(0);
+
+        done();
+      });
+  });
+
+  it('should not render the data when no type provided', (done) => {
+    const context = {
+      params: {
+        dataTestId: 'some-data-identifier',
+        data: [
+          {
+            id: 'C14E1',
+            name: 'access prescribable items',
+          },
+          {
+            id: 'C14E2',
+            name: 'manage Formularies',
+          },
+          {
+            id: 'C14E13',
+            name: 'access Patient Record',
+          },
+        ],
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+        expect($('[data-test-id="some-data-identifier"]').length).toEqual(0);
+
+        done();
+      });
+  });
+
+  it('should not render the data when not provided', (done) => {
+    const context = {};
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        expect($('[data-test-id="some-data-identifier"]').length).toEqual(0);
+
+        done();
+      });
+  });
+
+  it('should render the epic id', (done) => {
+    const context = {
+      params: {
+        dataTestId: 'some-data-identifier',
+        data: [
+          {
+            id: 'C14E1',
+            name: 'access prescribable items',
+          },
+        ],
+        type: 'met',
+      },
+    };
+
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        expect($('[data-test-id="some-data-identifier"] ul li span').text().trim()).toEqual('(C14E1)');
+
+        done();
+      });
+  });
+
+  it('should render the tick icon for met types', (done) => {
+    const context = {
+      params: {
+        dataTestId: 'some-data-identifier',
+        data: [
+          {
+            id: 'C14E1',
+            name: 'access prescribable items',
+          },
+        ],
+        type: 'met',
+      },
+    };
+
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        expect($('[data-test-id="some-data-identifier"] ul li svg').hasClass('nhsuk-icon')).toEqual(true);
+        expect($('[data-test-id="some-data-identifier"] ul li svg').hasClass('nhsuk-icon__tick')).toEqual(true);
+        done();
+      });
+  });
+
+  it('should render the cross icon for not-met types', (done) => {
```

Comment:
Ah ok I see you have done this here 👍 
I would structure this test file bit differently
```
describe('view-epic-list') {
  describe('when type is met') {
    // all your met unit tests here. names, icon and classes
  }
  describe('when type is not-met') {
    // all your not-met unit tests here.
  }
}
```